packages/installer/cos.sh

```diff
@@ -735,30 +753,31 @@ ensure_dir_structure() {
 }
 
 luet_args() {
+    local args
     args="--enable-logfile --logfile /tmp/luet.log"
-    if [ -z "$VERIFY" ] || [ "$VERIFY" == true ]; then
+    if [ -z "$_VERIFY" ] || [ "$_VERIFY" == true ]; then
```

Comment:
Those can be specified via config as well. It's a way to have defaults.

A specific prefix as well here makes sense